### ISSUE TITLE
Basic player prefab

### ISSUE THREAD

**Issue by RuralBrick** [RuralBrick/BitsAndBullets#3]
- [x] Movement
- [x] Collider
- [x] Aiming (+ visualization)

**Comment by RuralBrick** [comment]
- [x] Can fire from standstill towards last direction aimed at

**Comment by Alex-Hen** [comment]
Interesting note: If we want the rotation to snap instantly to the direction of motion, then the transform option seems like our best bet.  However, this causes some interesting clipping when you try to rotate into a wall.  I tried to use torque and rigidbody rotation to resolve this, but the torque turns the sprite into the worlds fastest figit spinner and also adds some lag during the rotation, and the rigidbody rotation has even more clipping.

TLDR: We can resolve rotation issues by choosing a circular sprite.

**Comment by RuralBrick** [comment]
@Alex-Hen I agree that using a circular sprite (+ directly manipulating rotation) works best.
